.circleci/config.yml

```diff
@@ -67,6 +195,19 @@ workflows:
   version: 2
   build-test-and-deploy:
     jobs:
+      - build:
+          filters:
+            tags:
+              ignore: /.*/
```

Comment:
What is this doing?

Reply:
I nabbed this from the template repo, it basically skips these checks when we push a git tag to the repo. As we don't have / use tags, this is probably unecessary noise